public/app/features/explore/LogsContainer.tsx

```diff
@@ -33,8 +34,12 @@ interface LogsContainerProps {
   loading: boolean;
 
   logsHighlighterExpressions?: string[];
-  logsResult?: LogsModel;
-  dedupedResult?: LogsModel;
+  logRows?: LogRowModel[];
+  logsMeta?: LogsMetaItem[];
+  logsSeries?: GraphSeriesXY[];
+  dedupedRows?: LogRowModel[];
+  hasUniqueLabels: boolean;
```

Comment:
A bit unclear why this one is not optional, and the other props are.

Reply:
Will change it to optional too to. Makes more sense for the cases when we do not have any logsResult.

Reply:
So seems that after the log row details we don't this variable anywhere so it goes away. Will take a look whether it can be removed from the LogsModel altogether in a separate PR.